ujson.go

```diff
@@ -6,17 +6,20 @@ import (
 )
 
 type JSON struct {
-	root interface{}
+	Root interface{}
```

Comment:
Sounds good. I will close this PR and create new one for the unicode support and one when I rework the Array api